tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -317,6 +322,123 @@ stages:
           artifactName: 'perf-test-outputs_memory-usage'
         condition: succeededOrFailed()
 
+  # Performance unit tests - customData
```

Comment:
NOTE: The entire `perf_unit_tests_customData` stage referred to `perf_unit_tests_memory` and `executionTestPackages` of `perf_unit_tests_runtime`.

Reply:
I was just suggesting that I pretty much used `perf_unit_tests_memory` as the template. And wanted to make sure that `perf_unit_tests_customData` is written correctly.

Reply:
Got it. Yeah, using the other one as template is fine, and I went through the new stage and everything that needed to be adjusted seemed correct.